src/main.rs

```diff
@@ -348,6 +330,41 @@ fn find_ram_in_linker_script(linker_script: &str) -> Option<MemoryEntry> {
     None
 }
 
+fn arithmetic_op(line: &str) -> u64 {
+    let segments: Vec<&str> = line.split('+').map(|s| s.trim().trim_end()).collect();
+    // if some numbers are hex we convert here
+
+    let mut total_length = 0;
+    for segment in segments {
+        println!("current segment {}", segment);
```

Comment:
Is this meant to be kept inside the actual output, or is it for debugging purposes? Same for the other print statements.